portals/devportal/src/main/webapp/source/src/app/components/Apis/Chat/Header.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable max-len */
```

Comment:
Better to split the code lines to be compatible with max-len instead of disabling it.

Reply:
And license header?